### PR TITLE
Requesting to pull for About page

### PR DESCRIPTION
**Pull request by AngRG26**
Developed 'Abus us' Page.

Added scroll to top function with buttons.

User can navigate to content by clicking arrows in sections.

Used couple of photos

### CODE REVIEW

**Comment by AngRG26** [comment]
@Zilola 
Hey Zilola, please review my code and try running it.
It seems like our website's content looks bit weird depends on screen resolution.

Do you know how to make screen look same to any resoultion?

**Comment by Zilola** [comment]
Hi @chlwjdtn15, yes I will have a look tomorrow. Please can you add in the description the screenshot of the page.

**Comment by AngRG26** [comment]
@Zilola 
Hi Zilola, in fact,the home page works fine with my resolution but on Elena's it does not.

@Loran-l 
Elena, could you post screenshot of our About us page on your screen?

**Comment by HyunSeongLee** [comment]
Jake, looks good this current About page and the scrolling option

**Comment by AngRG26** [comment]
MicrosoftTeams-image (1)
MicrosoftTeams-image

@Zilola, this is how our About us page shows on Elena's screen. It show bit off.

**Comment by Zilola** [comment]
Hi guys, Overall it looks great, good job.
Need some work:
- Header and Footer should match with the home page (After an addressed review on my ticket, Elena will merge my updates and after you can fix it) 
- Concept and Key Flow - the quality is not good. Can we change it?
- Class diagram -blurry, not able to read.
- conecptualModel.2340b7eb.JPG picture - delete or use html/css to create the same title with concepts 
- Need some work on styling please check PR for comments

File: src/components/About/About.css [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+.contentbox {
+    position: relative;
+    top: 10%;
+    left: 50%;
+    margin-bottom: 3%;
+    transform: translate(-50%, 10%);
+    box-sizing: border-box;
+    background: #fff;
+    border-radius: 4px;
+    box-shadow: 0 5px 15px rgba(0,0,0, .5);
```
Comment (Zilola) [inline src/components/About/About.css]: **change** box-shadow: 
    box-shadow: 0 5px 15px rgba(146, 117, 117, 0.1);
    **or** delete the shadow

Comment (AngRG26) [reply src/components/About/About.css]: changed

**Comment by chlwjdtn15** [comment]
> Hi guys, Overall it looks great, good job.
> Need some work:
> 
> * Header and Footer should match with the home page (After an addressed review on my ticket, Elena will merge my updates and after you can fix it)
> * Concept and Key Flow - the quality is not good. Can we change it?
> * Class diagram -blurry, not able to read.
> * conecptualModel.2340b7eb.JPG picture - delete or use html/css to create the same title with concepts
> * Need some work on styling please check PR for comments

**Pull request merged**